.changeset/sixty-items-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': minor
+---
+
+Added paragraph style
```

Comment:
It's a detail, but we have a convention to add a period at the end of a changeset message: 
```suggestion
Added paragraph style.
```